src/telegram/webhook.py

```
@@ -606,7 +606,7 @@ async def _cmd_freestyle(ctx: SlashCtx) -> None:
     if pipeline_preview == "repo":
         await send_message(
             ctx.chat_id,
-            "ℹ️ `/freestyle` doesn't apply to repo URLs yet\nRuning standard analysis.",
+            "ℹ️ `/freestyle` doesn't apply to repo URLs yet\nRunning standard analysis.",
```

Ruff (0.16.1)

[warning] String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)